bundles/org.openhab.binding.broadlink/README.md

```diff
@@ -0,0 +1,93 @@
+
+## Supported Things
+
+| Thing ID   | Description                                                             |
+|------------|-------------------------------------------------------------------------|
+| a1         | Broadlink A1 multi sensor                                               |
+| mp1        | Broadlink MP1 WiFi Smart Power Strip (4 sockets)                        |
+| mp1_1k3s2u | Broadlink MP1 1K3S2U WiFi Smart Power Strip (3 sockets, 2 USB)          |
+| mp2        | Broadlink MP2 WiFi Smart Power Strip (3 sockets, 3 USB)                 |
+| sp1        | Broadlink SP1 WiFi Smart Socket                                         |
+| sp2        | Broadlink SP2 WiFi Smart Socket with night light                        |
+| sp3        | Broadlink SP3/Mini WiFi Smart Socket with night light                   |
+| sp3s       | Broadlink SP3s WiFi Smart Socket with Power Meter                       |
+| rm         | Broadlink RM WiFI IR Transmitter                                        |
+| rm2        | Broadline RM2/Pro WiFi IR/RF Transmitter with temperature sensor        |
+| rm3        | Broadlink RM3/Mini WiFi IR Transmitter                                  |
+| rm3q       | Broadlink RM3 WiFi IR Transmitter with Firmware v44057                  |
+| rm4        | Broadlink RM4 WiFi IR Transmitter with temperature and humidity sensors |
+
+## Discovery
+
+Devices in the above list that are set up and working in the Broadlink mobile app should be discoverable by initiating a discovery from the OpenHAB UI.
+
+> The `Lock Device` setting must be switched off for your device via the Broadlink app to be discoverable in openHAB.
+
+## Thing Configuration
+
+| Name                | Type    | Default       | description                                                                       |
+|---------------------|---------|---------------|-----------------------------------------------------------------------------------|
+| ipAddress           | String  |               | Sets the IP address of the Broadlink device                                       |
+| staticIp            | Boolean | true          | Enabled if your broadlink device has a Static IP set                              |
+| port                | Integer | 80            | The network port for the device                                                   |
+| macAddress          | String  |               | The device's MAC Address                                                           |
+| pollingInterval     | Integer | 30            | The interval in seconds for polling the status of the device                      |
+| retries             | Integer | 1             | The number of re-attempts for a request before the device is considered `OFFLINE` |
+| ignoreFailedUpdates | Boolean | false         | Is enabled, failed status requests put the device `OFFLINE`                       |
```

Comment:
Does it ignore (not go offline) if the value is true/enabled? This seems to be confusing to me as the description does not match how I would expect a control named that way to work.